CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.1]
+### Updated
```

Comment:
The changelog format we use has a set list of categories (see https://keepachangelog.com/en/1.0.0/), "Changed" being most appropriate here I think
```suggestion
### Changed
```